server/src/fixtures/react.js.txt

```diff
@@ -10,10 +10,17 @@ const Test = () => {
 
 const Test2 = () => {
   const { isEnabled , get} = usePrefab();
+  const [searchParams, setSearchParams] = useSearchParams();
+  // This doesn't match because it is being called on an object
+  const searchTerm = searchParams.get("q") ?? "";
 
-  const foo= (get("new-logo")) ;
+  const foo= (get("some-value"));
 
-  const ok = isEnabled("new-logo") || isEnabled("new-logo2") || isEnabled("new-logo3");
+  const ok = isEnabled("abc-logo") || isEnabled("abc-logo2") || isEnabled("abc-logo3");
```

Comment:
Not re-using the same keys throughout the file makes it easier to read test failure messages.